conda_smithy/configure_feedstock.py

```diff
@@ -102,10 +105,10 @@ def render_circle(jinja_env, forge_config, forge_dir):
 
 @contextmanager
 def fudge_subdir(subdir):
-    orig = conda_build.metadata.cc.subdir
-    conda_build.metadata.cc.subdir = subdir
+    orig = cc.subdir
+    cc.subdir = subdir
     yield
-    conda_build.metadata.cc.subdir = orig
+    cc.subdir = orig
```

Comment:
This did delete run_docker_build.sh for me.  I see that this PR is pretty similar to what you https://github.com/conda-forge/conda-smithy/commit/b5a52b761271e768e3207eac78ac9d3bd42263fa - I do not have an answer for why your try at it did not work.